crawl-ref/source/initfile.cc

```diff
@@ -1788,6 +1791,9 @@ void game_options::read_options(LineInput &il, bool runscript,
 
         trim_string(str);
 
+        content += str;
```

Comment:
`content += str + "\n";`

Reply:
You think it's better on one line than two? I mean I can do that... I just thought it _really_ didn't matter.